orchestra/static/orchestra/task/task.html

```diff
@@ -114,7 +114,8 @@ <h3 class="modal-title">{{command | capitalize}} Confirmation</h3>
     <div class="container-fluid">
```

Comment:
This is great! Just so we don't confuse new users, we'll want to make sure we create the group and add it to the sample workflow data:
* [Our docs](https://orchestra.readthedocs.io/en/latest/example_use.html#setup) list the accounts that are created
* [This script](https://github.com/b12io/orchestra/blob/main/journalism_workflow/load_sample_data.py) does the loading
* I think the fix is to `from django.contrib.auth.models import Group` at the top of the script and add something like
```
project_admins = Group.objects.get_or_create(name='project_admins') 
project_admins.user_set.add(user)
```
on [this line](https://github.com/b12io/orchestra/blob/db95c574cf8c3aab42f63932fc6ceb995c10634a/journalism_workflow/load_sample_data.py#L21)